cmd/kubeadm/app/apis/kubeadm/validation/validation_test.go

```diff
@@ -102,15 +102,12 @@ func TestValidateNodeRegistrationOptions(t *testing.T) {
 		criSocket      string
```

Comment:
please remove criSocket from this struct.
and then use `nro := kubeadm.NodeRegistrationOptions{Name: rt.nodeName, CRISocket: "/some/path"}` bellow.

Reply:
done